Jenkinsfile

```diff
@@ -140,6 +140,34 @@ def daos_packages_version(String distro) {
     error "Don't know how to determine package version for " + distro
 }
 
+def fault_test_tag() {
+    if (skip_stage('fault-test')) {
+        return ",-faults"
+    }
+    return ""
+}
+
+def release_candidate() {
+    if (sh(label: "Determine if building (a PR of) an RC",
+          script: "git diff-index --name-only HEAD^ | grep -q TAG && " +
+                  "grep -i '[0-9]rc[0-9]' TAG",
+          returnStatus: true)) {
+        return true
+    }
+    return false
+}
+
+def faults_enabled(String type) {
+    // if the fault_enabled pragma is false or it a release candidate; disable fault injection
+    if (! cachedCommitPragma(pragma: 'faults_enabled', def_val: 'true') || release_candidate()) {
+        return ""
+    }
+    if (type == "rpm") {
+        return "--with fault-injection"
```

Comment:
typo here (missing -)? should this be --with-fault-injection ?

Reply:
No.  `rpmbuild`'s `--with` switch takes an argument which is then interpreted by the specfile.